build/webpack.config.ts

```diff
@@ -121,8 +121,6 @@ const webpackConfig: any = {
       src: paths.packageSrc('react'),
       docs: paths.base('docs'),
     },
-    // Allows to avoid multiple inclusions of the same module
-    modules: [paths.base('node_modules')],
```

Comment:
https://github.com/ReactTraining/react-router/issues/6201#issuecomment-403291934

The reason is here, we don't need this anymore due updates in Webpack config